code/game/objects/machinery/mortar.dm

```diff
@@ -210,6 +210,14 @@
 		flick(icon_state + "_fire", src)
 		mortar_shell.forceMove(src)
 
+		if(istype(I, /obj/item/mortal_shell/howitzer))
+			GLOB.round_statistics.howitzer_shells_fired++
+			SSblackbox.record_feedback("tally", "round_statistics", 1, "howitzer_shells_fired")		
+
+		else if(istype(I, /obj/item/mortal_shell))
```

Comment:
This block already knows that I is a shell from 157, you can just have this line as an else.